cmd/main/main.go

```diff
@@ -25,48 +27,35 @@ import (
 	"google.golang.org/grpc/reflection"
 	"google.golang.org/protobuf/encoding/protojson"
 
-	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	grpc_zap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap"
-	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
-
 	"github.com/instill-ai/connector-backend/config"
+	database "github.com/instill-ai/connector-backend/pkg/db"
 	"github.com/instill-ai/connector-backend/pkg/external"
 	"github.com/instill-ai/connector-backend/pkg/handler"
 	"github.com/instill-ai/connector-backend/pkg/logger"
+	custom_otel "github.com/instill-ai/connector-backend/pkg/logger/otel"
```

Comment:
Please arrange the layout of the import section

Reply:
done